.github/ISSUE_TEMPLATE/general-extension.md

```diff
@@ -7,7 +7,7 @@ about: Issues pertaining to downloading, installing, or building the extension.
 Type: General
 **Input information below**
 
-**Please review existing issues and our documentation at https://github.com/Microsoft/vscode-cpptools/tree/master/Documentation prior to filing an issue.**
+**Please review [existing issues](https://github.com/Microsoft/vscode-cpptools/issues) and our documentation at https://code.visualstudio.com/docs/languages/cpp prior to filing an issue.**
```

Comment:
Does the language server template need to be updated too?

Reply:
The language server template has the link but it is currently commented out. Was the link left out for some reason?

Reply:
The commenting out is intentional because we want users to see the comment but not have it shown when they file the issue, i.e. it's not commented out to the user.

Reply:
On all templates, I added links to 
- existing issues
- docs
- FAQs

They are commented out as well.